lib/shopify_api/resources/fulfillment.rb

```diff
@@ -9,5 +9,26 @@ def order_id
     def cancel; load_attributes_from_response(post(:cancel, {}, only_id)); end
     def complete; load_attributes_from_response(post(:complete, {}, only_id)); end
     def open; load_attributes_from_response(post(:open, {}, only_id)); end
+
+    def self.create(attributes = {})
```

Comment:
I think `self.create` is synonymous to calling `new` then `save`. https://api.rubyonrails.org/v3.2.6/classes/ActiveResource/Base.html#method-c-create so we should be good by just overriding save.